src/edu/washington/escience/myria/accessmethod/JdbcAccessMethod.java

```diff
@@ -425,6 +430,8 @@ public static String typeToDbmsType(final Type type, final String dbms) {
         return "BIGINT";
       case STRING_TYPE:
         return "TEXT";
+      case BLOB_TYPE:
+        return "BYTEA";
```

Comment:
BYTEA is a postres data type we are looking for...
https://wiki.postgresql.org/wiki/BinaryFilesInDB
We don't expect to store larger than 1GB files -- and the interface remains quite nice if we use bytea.